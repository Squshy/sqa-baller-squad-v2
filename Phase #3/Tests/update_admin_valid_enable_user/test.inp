LOGIN
ADMINBOY
AdPass123
UPDATE
ENABLE
yo
Yes
logout
exit
