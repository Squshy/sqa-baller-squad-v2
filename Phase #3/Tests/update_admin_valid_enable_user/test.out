=========================================================
                        UPDATE
=========================================================
Would you like to enable or disable a user?: ENABLE

Which user would you like to enable?: DO NOT HURT ME

Do you want to enable this user? (Yes/No): Y
---------------------------------------------------------
DO NOT HURT ME's account has been enabled!
=========================================================